contracts/src/registry/ETHRegistry.sol

```diff
@@ -66,6 +69,10 @@ contract ETHRegistry is LockableRegistry, AccessControl {
         return tokenId;
     }
 
+    function setRenewalObserver(uint256 tokenId, address _observer) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
I think we want this to be callable by the owner of the name. We're following CEI pattern below, so untrusted observers should be safe.